src/components/filters/MeasureValueFilter/OperatorDropdown.tsx

```diff
@@ -0,0 +1,88 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from "react";
+import { injectIntl, InjectedIntlProps } from "react-intl";
+import classNames from "classnames";
+import capitalize = require("lodash/capitalize");
+import Button from "@gooddata/goodstrap/lib/Button/Button";
+import { string as stringUtils } from "@gooddata/js-utils";
+
+import OperatorDropdownBody from "./OperatorDropdownBody";
+import { getOperatorTranslationKey, getOperatorIcon } from "../../../helpers/measureValueFilterOperator";
+
+export interface IOperatorDropdownOwnProps {
+    onSelect: (operator: string) => void;
+    operator: string;
+}
+
+export type IOperatorDropdownProps = IOperatorDropdownOwnProps & InjectedIntlProps;
+
+interface IOperatorDropdownState {
+    opened: boolean;
+}
+
+export class OperatorDropdown extends React.PureComponent<IOperatorDropdownProps, IOperatorDropdownState> {
+    constructor(props: any) {
```

Comment:
I'd rather specify the `props` type as `IOperatorDropdownProps`. Or, since props are not used to initialize state, you could omit constructor as a whole and just use

```ts
public state: IOperatorDropdownState = {
    opened: false,
};
```

Reply:
Removed, the constructor has no use here.